### DISCUSSION TITLE
Corona Virus Exploration

### DISCUSSION THREAD

**Discussion by prithvishkr** [neherlab/covid19_scenarios#659] [General]
I am a python develope.I was working with lstm and time series forecasting earlier.I have used FbProphet() for stock prediction and it is highly efficient and analytic in nature. FbProphet() uses date and closing value as its input.Then it predicts expected value of days ahead based on trends and calculation. So i had this crazy idea where in i used corona virus dataset in this. I know it does not have a closing value but instead i chose number of deaths as closing value and got this result.

![cap1](https://user-images.githubusercontent.com/60787716/81421635-60d21700-916f-11ea-9d68-2896c14c9f63.PNG)
![df](https://user-images.githubusercontent.com/60787716/81421641-629bda80-916f-11ea-9392-4306d14948e4.PNG)

**Comment by nnoll** [comment]
It's tough to interpret what these plots are... Are you plotting the number of predicted COVID-19 infections over different time windows of varying granularity? I also must confess, I'm ignorant as to what an "lstm" is. Would you mind enlightening me?

**Reply by softlyixe** [reply]
I think the world leaders have a hand in this covid-19

**Comment by prithvishkr** [comment]
Lstm stands for long short term memory and I tried to analyse the death of people with respect to time. Actually my doubt is that is that I used a stock prediction library for predicting this so is this model better wrt to other models present

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Reply by wk849411933** [reply]
What does wrt mean?

**Comment by prithvishkr** [comment]
Date versus deaths prediction
wrt means with respect to

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
